Scenarios/secrets.md

```diff
@@ -0,0 +1,184 @@
+# Secrets
+
+The behavior of AL-Go for GitHub is very much controlled by settings and secrets.
+
+To learn more about the settings used by AL-Go for GitHub, please navigate to [Settings](settings.md).
+
+## Where are the secrets defined

```

Comment:
Somewhere here you can include a link to https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#about-secrets

Reply:
done